sdk/src/lib.rs

```diff
@@ -130,7 +136,9 @@ impl TestRustWorker {
         workflow_id: impl Into<String>,
         workflow_type: impl Into<String>,
         input: Vec<Payload>,
+        mut options: WorkflowOptions,
     ) -> Result<String, tonic::Status> {
+        options.task_timeout = options.task_timeout.or(self.task_timeout);
```

Comment:
This seems ... ok-ish? We want to keep the value present in `options` if there is one, otherwise use the value in `self.task_timeout`, which itself may be `None`.

Reply:
This makes sense. Honestly `task_timeout` shouldn't even be on the whole worker probably, at least not in the way it is currently - but all this is going to get the treatment when I give this stuff a polish for real release. So this makes perfect sense for now